### ISSUE TITLE
Elastics has stopped working for me (OS X Sierra)

### ISSUE THREAD

**Issue by karock** [tundrabot/Elastics#6]
Not entirely sure if it's due to OS X Sierra or because I've started working with VPC instances, both happened simultaneously. It seems that the d-ross branch no longer compiles with the latest XCode either. Keeps complaining about some dependencies or libraries not being found. I tried to install them in the right places (OpenSSL, etc.) but it didn't work... ObjC isn't my area of expertise so maybe it's something easy to fix for someone who knows what they're doing.

**Comment by okcoker** [comment]
Same. Some errors from the console:

```
Elastics[70463]: ElasticsAppDelegate.m:1086 -[ElasticsAppDelegate disableRefreshTimer] disabling background refresh
Elastics[70463]: DataSource.m:283 -[DataSource requestDidFinishLoading:] 0.23s, kDataSourceCurrentRegionRefreshType
Elastics[70463]: DataSource.m:283 -[DataSource requestDidFinishLoading:] 0.04s, i-0a8d8df4750c49338
Elastics[70463]: objc[70463]: Cannot form weak reference to instance (0x10018eb80) of class NSCarbonMenuWindow. It is possible that this object was over-released, or is in the process of deallocation.
com.apple.xpc.launchd[1] (com.apple.ReportCrash[70578]): Endpoint has been activated through legacy launch(3) APIs. Please switch to XPC or bootstrap_check_in(): com.apple.ReportCrash
ReportCrash[70578]: Saved crash report for Elastics[70463] version 1.6 (160) to /Users/xxx/Library/Logs/DiagnosticReports/Elastics_2016-10-14-053654_xxx-5.crash
```

**Comment by devxoul** [comment]
I'll check this when I'm available. Thanks for reporting!

**Comment by karock** [comment]
@devxoul you ever get a chance to look into this?

**Comment by devxoul** [comment]
I haven't :( Actually I'm having very busy days refactoring the whole application of my company project.

Could you please how can I reproduce the bug?

**Comment by karock** [comment]
just have to open the list of servers and move your mouse up and down it. probably will even crash just looking at a single server eventually, but moving between them should crash it within 5-10 seconds.

**Comment by devxoul** [comment]
@karock, where is your Elastics.app from? AppStore?

**Comment by karock** [comment]
it was originally, but have since changed to a version compiled under the last OS. can't get the compilation to succeed since updating.

**Comment by devxoul** [comment]
@karock, could you please test with [this binary](https://dl.dropboxusercontent.com/u/69958027/Elastics.zip)? This is the compiled binary from the master branch. If it doesn't work, I'll check this when I become less busy.

**Comment by karock** [comment]
same result. crashed after moving mouse through the list of EC2 machines for a couple seconds. also the CPU graph never showed up.

**Comment by reinvented** [comment]
Since upgrading to macOS 10.12.2 I've not experienced these issues.

**Comment by karock** [comment]
I’m on 10.12.2 and am still seeing crashes. It might take a bit longer, but I can definitely get it to crash in under 30 seconds of messing with the menu.

**Comment by reinvented** [comment]
Yes, you're right: although I saw the frequency of crashing go down after the upgrade to macOS 10.12.2, it still crashes (also on 10.12.3).

**Comment by d-ross** [comment]
Hi - so I just updated to Sierra and am getting this crash quite a bit. First - to get my branch to build you need to install xcode 8.3 cmd line tools. Just do `xcode-select --install` from cmd line.

This crash is:

`objc[27025]: Cannot form weak reference to instance (0x1004d5ec0) of class NSCarbonMenuWindow. It is possible that this object was over-released, or is in the process of deallocation.`

No clue how to fix - will look when I can.
